common_features.mk

```diff
@@ -404,8 +404,12 @@ ifeq ($(strip $(SPACE_CADET_ENABLE)), yes)
   OPT_DEFS += -DSPACE_CADET_ENABLE
 endif
 
-
 ifeq ($(strip $(DIP_SWITCH_ENABLE)), yes)
   SRC += $(QUANTUM_DIR)/dip_switch.c
   OPT_DEFS += -DDIP_SWITCH_ENABLE
 endif
+
```

Comment:
Doesn't this need to be defaulted to yes to maintain the current behaviour?

Reply:
Nope,  previous version had no makefile support. It was added/enabled by adding an include.   Which .... was messy and problematic.